components/header.js

```diff
@@ -0,0 +1,72 @@
+import "./link-button/link-button.js";
+
+const template = document.createElement("template");
+template.innerHTML = `
+  <style>
+    :host {
+      position: fixed;
+      top: 0;
+      left: 0;
+      padding: 20px 0;
+      width: 100%;
+      border-bottom: 1px solid #D3D3D3;
+      background-color: rgba(255, 255, 255, 0.8);
+    }
+
+    header {
+      width: 80%;
+      margin: 0 auto;
+      justify-content: space-between;
+      display: flex;
+      align-items: center;
+    }
+
+    .buttons-container {
+      display: flex;
+    }
+
+    .logo {
+      display: flex;
+      gap: 10px;
+      align-items: center;
+      text-decoration: none;
+    }
+
+    .logo img {
+      width: 45px;
+      height: 22.5px;
+    }
+
+    .logo span {
+      font-size: 20px;
+      font-weight: 500;
+      color: black;
+    }
+
+    .logo img,
+    .logo span {
+      vertical-align: middle;
+    }
+  </style>
+  <header>
+      <a href="#" class="logo">
```

Comment:
URL을 `#`로 해도 작동은 하지만, 브라우저 주소줄에 불필요한 `#`가 붙게 됩니다.

```suggestion
      <a href="/" class="logo">
```

Reply:
refresh를 하지 않고 이동시키는게 목표라 #를 이용했는데 이런 주소줄에 #가 붙는게 많이 안좋긴 하겠군요.